index.html

```diff
@@ -0,0 +1,157 @@
+    Pricing
+    </header><!-- /header -->
+    <h4 class="myH">1: $5.99</h4>
+    <h4 class="myH">2-50: $4.49</h4>
+    <h4 class="myH">51+: $1.99</h4>
+  </div>
+</article>
+
+
+<article id="cardTwo">
+    <header id="header" class="dbg">
+       <h1 >Green Marble</h1>
+    </header><!-- /header -->
+
+  <div>
+    <img class="myImg" src="images/forestgreenmarble.jpg" alt="greenmarble">
+    <p>X-ray wellness diabetes. Migrane conditions red cross bones diabetes arteries brain. Activity tracking health mobile health 3D printing.</p>
+    <h4 class="dgb">Available</h4>
+  </div>
+
+  <div>
+    <header id="header" class="">
+     Specifications
+    </header><!-- /header -->
+    <h4 class="myH">size: 8" x 8"</h4>
+    <h4 class="myH">weight: 16 oz.</h4>
+    <footer>
+      <h6>specifications valid until 5/01/2016</h6>
+    </footer>
+  </div>
+
+  <div>
+    <header id="header" class="">
+    Pricing
+    </header><!-- /header -->
+    <h4 class="myH">1: $5.99</h4>
+    <h4 class="myH">2-50: $4.49</h4>
+    <h4 class="myH">51+: $1.99</h4>
+  </div>
+</article>
+
+
+<article id="cardThree">
+    <header id="header" class="dbg">
+       <h1>Dark Grey Marble</h1>
+    </header><!-- /header -->
+
+  <div>
+    <img class="myImg" src="images/darkgreymarble.jpg" alt="drkgreymarb">
+    <p>Housing train flying. Housing USA worldwide recommendations sailing. Gateway translator Vienna Australia money diary active lifestyle housing airBnB.</p>
+    <h4 class="dgb">Unavailable</h4>
+  </div>
+
+  <div>
+    <header id="header" class="">
+     Specifications
+    </header><!-- /header -->
+    <h4 class="myH">size: 8" x 8"</h4>
+    <h4 class="myH">weight: 16 oz.</h4>
+    <footer>
+      <h6>specifications valid until 5/01/2016</h6>
+    </footer>
+  </div>
+
+  <div>
+    <header id="header" class="">
+    Pricing
+    </header><!-- /header -->
+    <h4 class="myH">1: $5.99</h4>
+    <h4 class="myH">2-50: $4.49</h4>
+    <h4 class="myH">51+: $1.99</h4>
+  </div>
+</article>
+
+
+<article id="cardFour">
+    <header id="header" class="dbg">
+	     <h1>Brown Marble</h1>
+    </header><!-- /header -->
+
+  <div>
+    <img class="myImg" src="images/brownMarb.jpg" alt="brownmarble">
+    <p>Bandwidth niche market analytics ramen vesting period disruptive business plan mass market termsheet. MVP release infrastructure paradigm shift low hanging fruit churn rate long tail.</p>
+    <h4 class="dgb">Available</h4>
+  </div>
+
+  <div>
+    <header id="header" class="">
+	   Specifications
+    </header><!-- /header -->
+    <h4 class="myH">size: 8" x 8"</h4>
+    <h4 class="myH">weight: 16 oz.</h4>
+    <footer>
+      <h6>specifications valid until 5/01/2016</h6>
+    </footer>
+  </div>
+
+  <div>
+    <header id="header" class="">
+    Pricing
+    </header><!-- /header -->
+    <h4 class="myH">1: $5.99</h4>
+    <h4 class="myH">2-50: $4.49</h4>
+    <h4 class="myH">51+: $1.99</h4>
+  </div>
+</article>
+
+</body>
+</html>
+
```

Comment:
Convention is to leave a single blank line at the end of the file, other blank lines should be trimmed.